src/components/TemporaryProductPage/TemporaryProductPage.tsx

```diff
@@ -0,0 +1,254 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import styles from './styles.module.scss';
+import blankIcon from '../../assets/images/icons/favorites-blank.svg';
+import filledIcon from '../../assets/images/icons/favorites-filled.svg';
+import { ColorButton } from '../ColorButton';
+import { useLocation } from 'react-router-dom';
+import { COLORS } from '@/constants/colors';
+import { ParameterButton } from '../ParameterButton';
+import { getSeparetedCapacity } from '@/utils/getSeparetedCapacity';
+import { PrimaryButton } from '../PrimaryButton';
+import { PrimaryButtons } from '@/enums';
+import { useSelector } from 'react-redux';
+import { RootState } from '@/app/store';
+import { toggleClickedBuy } from '@/features/сart/сartSlice';
+import { FavoritesButton } from '../FavoritesButton';
+import { useCart } from '@/hooks';
```

Comment:
Regarding imports: our mutual colleague Anna said that first there must be external libraries => internal libraries => our files and styles.